content/en/serverless/step_functions/installation.md

```diff
@@ -185,6 +185,28 @@ For sample stacks and additional code examples, see [CDK Examples for Instrument
 [3]: https://github.com/DataDog/datadog-cdk-constructs
 [4]: /serverless/guide/step_functions_cdk
 {{% /tab %}}
+{{% tab "AWS SAM" %}}
+1. Ensure you have deployed the [Datadog Lambda Forwarder][1], and that you are using v3.130.0+. You might need to [update your Forwarder][1].
```

Comment:
```suggestion
1. Ensure you have deployed the [Datadog Lambda Forwarder][1], and that you are using v3.130.0 or later. You might need to [update your Forwarder][1].
```

Reply:
@cswatt Would you be able help update this instruction for all of the tabs?